src/common/modules/dropdown/currency-select-dropdown/CurrencySelectDropdown.vue

```diff
@@ -6,45 +6,83 @@
                        class="currency-select-dropdown"
                        :class="{ 'print-mode': printMode }"
                        @select="handleSelectCurrency"
-    />
+    >
+        <template v-if="defaultMode"
+                  #default="{ item }"
+        >
+            <span>
+                <span>{{ item.label }}</span>
+                <p-badge v-if="item.badge"
+                         style-type="primary3"
+                         shape="round"
+                >{{ item.badge }}</p-badge>
+            </span>
+        </template>
+        <template v-if="defaultMode"
```

Comment:
todo: Line 10 and this line are the same...!

Reply:
Yes. . . Only slot name is different.
But I did not make a new component because the code is too easy and simple.

Reply:
Oh I understand :) Sorry for misunderstanding